content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -5,10 +5,22 @@ weight: 80
 keywords: [security,authentication,migration]
 ---
 
-This task shows how to migrate your existing Istio services' traffic from plain text to mutual TLS without breaking live traffic.
-
-In practice, a mesh consists of both Istio services (with Envoy sidecars) and services without Envoy sidecar(called "legacy" below for simplicity).
-A legacy service can't use Istio issued key/certificate to send mutual TLS traffic. We want to enable mutual TLS incrementally, safely.
+This task shows how to migrate your existing Istio services' traffic from plain
+text to mutual TLS without breaking live traffic.
+
+Imagine you have many services talking to each other and you want to
+gradually migrate them to Istio. During the migration, some services have Envoy
+sidecars while some do not. For a service with a sidecar, if you enable
+mutual TLS on it, the connections from legacy clients (i.e., clients without
+Envoy) will break since they do not have Envoy sidecars and client certificates.
```

Comment:
will break?  could you be a little more descriptive such as "will lose communication" or "will cease to communicate in the Istio mesh"

Reply:
done.